website/docs/index.html.markdown

```diff
@@ -37,6 +37,7 @@ resource "datadog_timeboard" "default" {
 ## Argument Reference
 
 The following arguments are supported:
-* `api_key` - (Required) Datadog API key. This can also be set via the `DD_API_KEY` environment variable.
-* `app_key` - (Required) Datadog APP key. This can also be set via the `DD_APP_KEY` environment variable.
+* `api_key` - (Optional) Datadog API key. This can also be set via the `DD_API_KEY` environment variable.
```

Comment:
This is a nit, but maybe something like:
```suggestion
* `api_key` - (Required unless `validate` is false) Datadog API key. This can also be set via the `DD_API_KEY` environment variable.
```
(For the app key as well) Let me know what you think